src/neo/SmartContract/NefFile.cs

```diff
@@ -110,20 +111,19 @@ public void Deserialize(BinaryReader reader)
         /// <returns>Return checksum</returns>
         public static uint ComputeChecksum(NefFile file)
         {
-            using (var ms = new MemoryStream())
-            using (var wr = new BinaryWriter(ms))
-            {
-                file.Serialize(wr);
-                wr.Flush();
+            using var ms = new MemoryStream();
+            using var wr = new BinaryWriter(ms, Encoding.UTF8, false);
 
-                // Read header without CRC
+            file.Serialize(wr);
+            wr.Flush();
 
-                Span<byte> buffer = stackalloc byte[HeaderSize - sizeof(uint)];
-                ms.Seek(0, SeekOrigin.Begin);
-                ms.Read(buffer);
+            // Read header without CRC
 
-                return BitConverter.ToUInt32(buffer.Sha256(), 0);
-            }
+            Span<byte> buffer = stackalloc byte[HeaderSize - sizeof(uint)];
+            ms.Seek(0, SeekOrigin.Begin);
+            ms.Read(buffer);
+
+            return BitConverter.ToUInt32(buffer.Sha256().Sha256(), 0);
```

Comment:
`Hash256` is accessible here? https://github.com/neo-project/neo/blob/1af936929093721ec274167829761ddba825c86c/src/neo/Cryptography/Crypto.cs#L14-L17

Reply:
Good! and done :)

Reply:
? it's still `buffer.Sha256().Sha256()`.